### PR TITLE
Resolves RTMD-32: Update the ci build steps to run acceptance tests a…

### PR DESCRIPTION
**Pull request by JoeChapman**
…gainst the service

### CODE REVIEW

**Comment by JoeChapman** [comment]
@easternbloc

File: ci_build.sh [hunk 1]
```diff
@@ -60,25 +60,13 @@ CONFIG_FILE=config_ci_build.yml
...
-     -e "CONFIG_FILE=${CONFIG_FILE}" \
-     ${ACCEPTANCE_TAG}
 
-# Pause before running tests
-sleep 10
-if docker run -i --rm=true \
-     --link ${APP_HOST}:${APP_HOST} \
-     -e "CONFIG_FILE=${CONFIG_FILE}" \
-     ${ACCEPTANCE_TAG} ; then
+if npm run test:acceptance ; then
```
Comment (easternbloc) [inline ci_build.sh]: I'd rather this if was in square brackets and the then was on the next line.

Comment (JoeChapman) [inline ci_build.sh]: I've added the square braces, but left `then` on the same line because that is the style used throughout this file

**Comment by JoeChapman** [comment]
@easternbloc

**Pull request merged**